grammars/README.md

```diff
@@ -9,6 +9,7 @@ an index.
 
 - [Ace](https://ace.c9.io/) — supported. The grammar is upstream
   ([prql_highlight_rules.js](https://github.com/ajaxorg/ace/blob/master/src/mode/prql_highlight_rules.js)).
+  See the [demo](https://prql-lang.org/ace-demo).
```

Comment:
I didn't realize this was a top-level page. Could we put it down a level to indicate that it's not a sibling of the key pages? Maybe a `content/demos/ace-demo.html` page?

Reply:
Sounds good.